src/cmap/connection.ts

```diff
@@ -1078,50 +951,17 @@ export class ModernConnection extends TypedEventEmitter<ConnectionEvents> {
    *
    * This method does nothing if the connection is already closed.
    */
-  private cleanup(force: boolean, error?: Error): void {
+  private cleanup(error?: Error): void {
     if (this.closed) {
       return;
     }
 
-    this.closed = true;
-
-    const completeCleanup = () => {
-      for (const op of this[kQueue].values()) {
-        op.cb(error);
-      }
-
-      this[kQueue].clear();
-
-      this.emit(Connection.CLOSE);
-    };
-
-    this.socket.removeAllListeners();
-    this[kMessageStream].removeAllListeners();
-
-    this[kMessageStream].destroy();
-
-    if (force) {
-      this.socket.destroy();
-      completeCleanup();
-      return;
-    }
-
-    if (!this.socket.writableEnded) {
-      this.socket.end(() => {
-        this.socket.destroy();
-        completeCleanup();
-      });
-    } else {
-      completeCleanup();
-    }
+    this.socket.destroy();
```

Comment:
question: why did we remove the option to gracefully close our sockets (`force: false`)?